scripts/update-extension.sh

```diff
@@ -33,3 +33,4 @@ else
       --version $EXTENSION_TAG \
       --configuration-settings-file $EXTENSION_SETTINGS
 fi
+#test
```

Comment:
is this here on purpose?

Reply:
Yeah this is just temporary to trigger the upgrade job, I'll remove this when it's done.